kci_build

```diff
@@ -401,6 +403,16 @@ class cmd_push_kernel(Command):
         install = kernelci.build.Step.get_install_path(args.kdir, args.output)
         meta = kernelci.build.Metadata(install)
         publish_path = meta.get('bmeta', 'kernel', 'publish_path')
+        defconfig = meta.get('bmeta', 'kernel', 'defconfig')
+        # gki_defconfig is exceptional and generate hundreds of megabytes
+        # several files, which require compression
+        # to workaround current upload limits
+        if (defconfig == "gki_defconfig"):
+            kernel_fname = meta.get('bmeta', 'kernel', 'image')
+            compressed_artifacts = ["logs/kernel.log"]
+            compressed_artifacts.append("kernel/"+kernel_fname)
```

Comment:
A bit more Pythonesque:
```suggestion
            compressed_artifacts.append('/'.join(["kernel", kernel_fname]))
```
Or `os.path.join('kernel', kernel_fname)` since it's a path.
Or `f"kernel/{kernel_fname}"`.
Also `kernel_fname` doesn't seem to clear, it's the kernel image name.

Reply:
Updated